### ISSUE TITLE
Feedback on mini_ex2 by Maiken

### ISSUE THREAD

**Issue by ghost** [nannanoermark/miniexes_real#4]
Your program is the face of a clown, which allows the user to place his mouth anywhere on the screen with the mouthPressed function. You mention that the mouthPressed function was the most challenging thing to you, but it seems to work like intended, so you did a good job with that. Other than that, the program mostly consists of simple shapes, which is well in line with the guidelines for mini ex 2. I do, however, think you could challenge yourself to use even more of the new stuff we learn next time. For example, you could make a condition so that mousePressed would only work if pressed within the coordinates of the head. You mention you wanted it to be a childish type of fun, and it works very well with the clown looking silly when the mouth is unusual places on the face. But at least for me personally, when the ellipse is no longer connected to the face in any way, i can't see it as a mouth anymore, so the condition could be beneficial here. 

To sum up, i think you did a very good job at making a fun, simple program according to the guidelines, but you could challenge yourself to use more of the new things we learn next time.
